src/Microsoft.PowerShell.Commands.Management/cimSupport/cmdletization/cim/cimQuery.cs

```diff
@@ -314,11 +314,8 @@ public override void FilterByAssociatedInstance(object associatedInstance, strin
         /// <param name="optionValue"></param>
         public override void AddQueryOption(string optionName, object optionValue)
         {
-            if (string.IsNullOrEmpty(optionName))
-            {
-                throw new ArgumentNullException(nameof(optionName));
-            }
-
+            ArgumentException.ThrowIfNullOrEmpty(optionName);
+            
```

Comment:
```suggestion
```

Reply:
My personal preference is to separate the method calls with a blank line when the type differs.

Reply:
I like it, I'll update my PR following your style 👍

Reply:
I think it's better to group them together as they are doing the same kind of things. For example, we group `Dbg.Assert(xxx)` together at the beginning of some methods, even though they do different checks.